src/subcommands/projects.ts

```diff
@@ -118,13 +118,20 @@ async function showProject(args: Args): Promise<void> {
     );
     return Deno.exit(1);
   }
+  const organizationName = project.organization.name
+    ? magenta(project.organization.name)
+    : `${
+      magenta(
+        (await api.getOrganizationById(project.organization.id))!.members[0]
+          .user
+          .name,
+      )
```

Comment:
Are we certain that this non-null assertion is always safe?

Reply:
Null means the organization has been deleted. Technically it could happen If the organization is deleted after the project is fetched and before the organization is fetched, but that's a negligible race IMO.

Reply:
Oh I see that kinda makes sense - maybe worth adding a comment explaining that? Whenever I see non-null assertions, as XXX, any, or something like these, I become very watchful 👀